### ISSUE TITLE
Keep selected track when open external player

### ISSUE THREAD

**Issue by A2va** [jellyfin/jellyfin-android#727]
**Describe the feature you'd like**

For now when a video is played with a external player, the subtitle/audio track are not corresponded to those that are selected in Jellyfin app.
 I wanted to implement it by me and I look at the code, in `ExternalPlayer.kt` and `playMediaSource` function. The function `getVideoStreamUrl` is used and accept a `subtitleStreamIndex` as well as audioStreamIndex and videoStreamIndex arguments.
I modified the code as follows:
```kotlin
 videosApi.getVideoStreamUrl(
    itemId = source.itemId,
    static = true,
    mediaSourceId = source.id,
    playSessionId = source.playSessionId,
    subtitleStreamIndex = source.selectedSubtitleStream?.index
)
```
Sadly the behaviour is the same as previous, So is this a bug in the Jellyfin server ?

**Comment by Nalsai** [comment]
This seems to be related to #595. When transcoding, the track selection is ignored and the default is always used.

**Comment by Maxr1998** [comment]
Yep, I was already working on fixing that now that the quality selection is here. Unfortunately my draft is still very broken and I didn't have time to debug it yet. Soon!
